core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -433,16 +433,20 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
       this.activeSuggestion.target_id, this.activeSuggestionId,
       AppConstants.ACTION_ACCEPT_SUGGESTION,
       reviewMessageForSubmitter, this.finalCommitMessage,
-      this.resolveSuggestionAndUpdateModal.bind(this),
+      () => {
+        this.alertsService.clearMessages();
+        this.alertsService.addSuccessMessage('Suggestion accepted.');
+        this.resolveSuggestionAndUpdateModal();
+      },
       (errorMessage) => {
-        this.rejectAndReviewNext(`Invalid Suggestion: ${errorMessage}`);
+        const message = `Invalid Suggestion: ${errorMessage}`;
         this.alertsService.clearWarnings();
-        this.alertsService.addWarning(
-          `Invalid Suggestion: ${errorMessage}`);
+        this.alertsService.addWarning(message);
+        this._reject(message);
```

Comment:
I'm not sure it makes sense to automatically reject the suggestion if there was a failure during accepting. If the original intent was to accept the suggestion, but there was a backend error, I don't think we should reject it, since atm that will force the user to resubmit the suggestion all over again. Instead, the reviewer should edit the suggestion to correct any errors and try accepting again.

Reply:
Done.